packages/wdio-browserstack-service/src/cleanup.ts

```diff
@@ -27,6 +29,14 @@ export default class BStackCleanup {
             const error = err as string
             BStackLogger.error(error)
         }
+
+        try {
+            if (process.argv.includes('--performanceData')) {
+                PerformanceTester.uploadEventsData()
```

Comment:
This is an async function, do we need to `await` it?

Reply:
if not, we should remove the try catch

Reply:
Thanks, fixed. Added await